lib/pacemaker-pe_status.pc.in

```diff
@@ -13,4 +13,4 @@ Requires.private: glib-2.0 libxml-2.0 @PC_NAME_CURSES@
 Conflicts:
 Cflags:           -I${includedir}
 Libs:             -L${libdir} -l${sub}
-Libs.private:     @PC_LIBS_CURSES@
```

Comment:
Awesome, tumbleweed finally stopped complaining :)

Of course crm_mon will still be broken there until they address the pcre issue, but at least the cluster will work

Reply:
Ah, I was wondering why it all suddenly went green.